### PR TITLE
Portfolio-project

### PR DESCRIPTION
**Pull request by micheaol**
In this pull request I add:
1. Make sure the website displays correctly on desktop screens.
2. I used a media query to make the website display on desktop screens.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -49,177 +51,197 @@ <h1 class="intro-title">I’am a Software Developer</h1>
...
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                </div>
+                <div class="tech-lang">
+                    <ul>
+                        <li>css</li>
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li id="last-child">Ruby</li>
+                    </ul>
+                    <button type="button" id="post-btn">See Project</button>
```
Comment (karvel-code) [inline index.html:67]: - [ ] Please ensure that the See project button directs a user to the live version of your project or your git hub repository. You can do this by using the `<a>` tag and styling it based on the Figma design.

File: index.html [hunk 2]
```diff
@@ -49,177 +51,197 @@ <h1 class="intro-title">I’am a Software Developer</h1>
...
-            <div class="job-tech-lang">
-                <ul class="job-tech-lang-ul">
-                    <li class="job-item job-one">html</li>
-                    <li class="job-item job-one">bootstrap</li>
-                    <li class="job-item job-one">Ruby</li>
-                </ul>
+            <div class="multi-post-inner">
+                <h1 class="post-title">Multi-Post Stories</h1>
+                <div class="post">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
```
Comment (karvel-code) [inline index.html:58]: - [ ] It is advisable that your sentence only takes up about 3-4 lines. In your case it's up to five, please mend that. You can improve on it by either reducing the font size slightly or increasing the section's width slightly.

File: index.html [hunk 3]
```diff
@@ -49,177 +51,197 @@ <h1 class="intro-title">I’am a Software Developer</h1>
...
+                        </div>
+                        </div>
+                        <button type="button" class="see-jobs-btn show-in-desktop">See Project</button>
+                </div>
+                <div class="job-item-with-btn">
+                    <div class="job-item-two">
+                        <h1 class="job-post-title">Profesional Art Printing Data</h1>
+                        <div class="job-post">
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        </div>
```
Comment (karvel-code) [inline index.html:93]: - [ ] Ensure that the texts in all your project cards are only taking about 2-3 lines only.

**Review by karvel-code — CHANGES_REQUESTED** [review]
Hi @micheaol , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Your code look quite professional!
- You have followed git flow.
- All your linter checks are passing.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Adding the screen shot for the desktop version to your README file.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ZeenatLawal — APPROVED** [review]
Hi @micheaol, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**